app/src/components/domain/ActiveOperationMap/index.tsx

```diff
@@ -48,13 +47,13 @@ import {
     DEFAULT_MAP_PADDING,
     DURATION_MAP_ZOOM,
 } from '#utils/constants';
-import { adminFillLayerOptions } from '#utils/map';
 import type {
     GoApiResponse,
     GoApiUrlQuery,
 } from '#utils/restRequest';
 import { useRequest } from '#utils/restRequest';
 
+import GlobalMap, { AdminZeroFeatureProperties } from '../GlobalMap';
```

Comment:
```suggestion
import GlobalMap, { type AdminZeroFeatureProperties } from '../GlobalMap';
```

Reply:
I have added a linter rule for this issue.